geomstats/geometry/spd_matrices.py

```diff
@@ -108,6 +109,105 @@ def random_tangent_vec_uniform(self, n_samples=1, base_point=None):
 
         return tangent_vec
 
+    @staticmethod
+    def submersion(point):
```

Comment:
This shouldn't be in the SPD class, otherwise, as you said you can only have one quotient strucutre with total space SPD